numpy/core/_internal.py

```diff
@@ -631,4 +631,17 @@ class TooHardError(RuntimeError):
     pass
 
 class AxisError(ValueError, IndexError):
-    pass
+    """ Axis supplied was invalid. """
```

Comment:
I put it here in #8584 because it needs to be imported from C code, and that's what `_internals` seems to be used for. Is it safe to load `_globals.py` from C?